.github/scripts/latest_version_release.sh

```diff
@@ -28,47 +26,40 @@ pushd "$GITHUB_WORKSPACE/charts/$chart_dir"
     chart_version=$(grep '^version:[[:space:]][[:digit:]]' Chart.yaml | awk '{print $2}') || die "Failed to retrieve current chart version"
     bump_version "${chart_version}" "${major_minor_patch}"
     new_chart_version=$new_version && echo "The new Chart version is: ${new_chart_version}"
-    sed -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
+    sed  -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
```

Comment:
```suggestion
    sed -i "s/version:.*/version: ${new_chart_version}/g" Chart.yaml
```

Reply:
why is that space needed?

Reply:
it's not I've removed it in the suggestion

Reply:
updated